src/script/addRemove.js

```diff
@@ -68,7 +68,7 @@ class Store {
   static modifyTask(elem) {
     const tasks = Store.getTasks();
     console.log(elem.id);
```

Comment:
- [ ] For javascript best practices I think it would be nice if you remove all `console.log` in your project.

Reply:
Asked change implemented